test/metabase/api/common/internal_test.clj

```diff
@@ -1,9 +1,123 @@
 (ns metabase.api.common.internal-test
-  (:require [clojure.test :refer :all]
-            [medley.core :as m]
-            [metabase.api.common.internal :as internal]
-            [metabase.config :as config]
-            [metabase.util :as u]))
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as http]
+   [clojure.test :refer :all]
+   [clojure.test :refer :all]
+   [clout.core :as clout]
+   [compojure.core :refer [POST]]
+   [medley.core :as m]
+   [metabase.api.common :as api]
+   [metabase.api.common.internal :as internal]
+   [metabase.api.common.internal :as api.internal]
+   [metabase.config :as config]
+   [metabase.server.middleware.exceptions :as mw.exceptions]
+   [metabase.server.middleware.json :as mw.json]
+   [metabase.server.middleware.misc :as mw.misc]
+   [metabase.server.middleware.security :as mw.security]
+   [metabase.util :as u]
+   [ring.adapter.jetty :as jetty]
+   [malli.error :as me]
+   [malli.generator :as mg]
+   [malli.util :as mu])
+(:import clojure.lang.ExceptionInfo))
+
+(def TestAddress
+  [:map
+   {:title "Address"}
+   [:id string?]
+   ;; TODO it is possible to coerce things like this automatically:
+   ;; [:tags [:set keyword?]]
+   [:tags [:vector string?]]
+   [:address
+    [:map
+     [:street string?]
+     [:city string?]
+     [:zip int?]
+     [:lonlat [:tuple double? double?]]]]])
+
+(def ClosedTestAddress
+  (mu/closed-schema TestAddress))
+
+(api/malli-defendpoint POST "/post/any" [:as {body :body :as request}]
+                       {:status 200 :body body})
```

Comment:
Shouldn't these be getting indented 2 spaces, like the old version of `defendpoint`? Since it's the body of a macro

Reply:
Yeah, that was getting caused by a lack of the `def` prefix.